### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ajmeese7**
Updated link as the redirect page requested

### CODE REVIEW

**Review by vsemozhetbyt — APPROVED** [review]
Thank you!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 The Code of Conduct explains the *bare minimum* behavior
 expectations the Node Foundation requires of its contributors.
-[Please read it before participating.](https://github.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md)
+[Please read it before participating.](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
```
Comment (Tiriel) [inline CONTRIBUTING.md:7]: I think the CoC has just been moved in the admin repo: https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md
https://github.com/nodejs/admin/pull/25

Comment (daxlab) [reply CONTRIBUTING.md:7]: @ajmeese7 can you please update this pr ?

Comment (ajmeese7) [reply CONTRIBUTING.md:7]: @daxlab I accidentally deleted the merging branch so I'm not sure how to do that

**Comment by fhemberger** [comment]
Fixed in ab77c33e714408553ad71dc6f9e3048b55d3c558.
Sorry, couldn't update your commit as you deleted the branch.

**Pull request closed without merging**